Hangfire.MySql/Install.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE `[tablesPrefix]Job` (
   `CreatedAt` datetime(6) NOT NULL,
   `ExpireAt` datetime(6) DEFAULT NULL,
   PRIMARY KEY (`Id`),
-  KEY `IX_Job_StateName` (`StateName`)
+  KEY `IX_Job_StateName[tablesPrefix]` (`StateName`)
```

Comment:
Could you please move it in front of table name, like `IX_[tablesPrefix]Job_StateName`